### PR TITLE
Positional parameters

### PR DESCRIPTION
**Pull request by levydsa**

### CODE REVIEW

File: libsql/src/main/java/tech/turso/libsql/Connection.java [hunk 1]
```diff
@@ -12,25 +13,44 @@ public class Connection implements AutoCloseable {
     }
 
     public void execute(String sql) {
-        Log.i("conn", "executing " + sql);
         if (nativePtr == 0) throw new RuntimeException("Attempted to execute a null connection");
+        Log.i("conn", "executing " + sql);
```
Comment (haaawk) [inline libsql/src/main/java/tech/turso/libsql/Connection.java:17]: Do we want to keep this?

File: libsql/src/main/java/tech/turso/libsql/Connection.java [hunk 2]
```diff
@@ -12,25 +13,44 @@ public class Connection implements AutoCloseable {
...
+
         nativeExecute(nativePtr, sql);
     }
 
     public void query(String sql) {
-        Log.i("conn", "querying " + sql);
         if (nativePtr == 0) throw new RuntimeException("Attempted to query a null connection");
+        Log.i("conn", "querying " + sql);
```
Comment (haaawk) [inline libsql/src/main/java/tech/turso/libsql/Connection.java:24]: Do we want to keep this?

File: libsql/src/main/java/tech/turso/libsql/Connection.java [hunk 3]
```diff
@@ -12,25 +13,44 @@ public class Connection implements AutoCloseable {
...
+
         nativeQuery(nativePtr, sql, new byte[] {});
     }
 
     public void query(String sql, Map<String, Value> params) {
+        if (nativePtr == 0) throw new RuntimeException("Attempted to query a null connection");
+        Log.i("conn", "querying " + sql);
```
Comment (haaawk) [inline libsql/src/main/java/tech/turso/libsql/Connection.java:31]: Do we want to keep this?

File: libsql/src/main/java/tech/turso/libsql/Connection.java [hunk 4]
```diff
@@ -12,25 +13,44 @@ public class Connection implements AutoCloseable {
...
                         .build()
                         .toByteArray();
-        Log.i("conn", "querying " + sql);
+
+        nativeQuery(nativePtr, sql, buf);
+    }
+
+    public void query(String sql, Value... params) {
         if (nativePtr == 0) throw new RuntimeException("Attempted to query a null connection");
+        Log.i("conn", "querying " + sql);
```
Comment (haaawk) [inline libsql/src/main/java/tech/turso/libsql/Connection.java:44]: Do we want to keep this?

**Review by haaawk — APPROVED** [review]

File: libsql/src/main/java/tech/turso/libsql/Connection.java (hunk 4 above)
Comment (levydsa) [reply libsql/src/main/java/tech/turso/libsql/Connection.java:44]: I removed the logs in the Kotlin version

**Pull request merged**